pkg/pb/pointerdb.proto

```diff
@@ -56,6 +56,7 @@ message Pointer {
   google.protobuf.Timestamp expiration_date = 7 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
 
   bytes metadata = 8;
+  bool piece_hashes_verified = 9;
```

Comment:
Could we possibly make this an integer, and return the number of piece hashes that were successfully verified? There might not be any great way to let the uplink know whether some of the hashes failed verification otherwise, and I think that "we stored your pointer, but not with as much redundancy as you think" is information that the uplink needs to know.

Reply:
Oh, oops. Not here, this doesn't get returned to the client. Can we add such an integer to SegmentCommitResponse, instead

Reply:
We can do something like that - the only issue is that we want to be able to distinguish pointers that were created after this change vs. before this change. And `0` is an ambiguous value for an integer in this case. Does it mean the pointer was created before this change or does it mean we successfully verified 0 hashes? Of course the latter is impossible if we end up storing the pointer, but I think it makes the purpose of this value less obvious.

Reply:
You're totally right, it's better to have a bare flag in the pointer. I had my structs mixed up.

Reply:
In case my second comment above got lost because it came seconds before yours, see above. :) I'm hoping for an int field in SegmentCommitResponse instead

Reply:
We could still add a field in `SegmentCommitResponse` if you think it would be worthwhile.

Reply:
Yes please. Good idea

Reply:
Done